### PR TITLE
Visual Update

### PR DESCRIPTION
**Pull request by ascott1**
This begins to bring DOCter more in line with current visual standards and adds a `brand_color` variable to easily swap the CFPB green with another color.

screen shot 2014-04-22 at 8 22 01 am

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -191,32 +211,62 @@ Layout
...
     margin: 0 auto;
-    padding: 0 20px;
+    padding-left: 20px;
+    padding-right: 20px;
 }
 
 header {
     width: 100%;
-    border-bottom: 4px solid #2CB34A;
+    border-bottom: 4px solid {{ site.brand_color }};
```
Comment (Scotchester) [inline assets/css/main.css:240]: I had no idea you could use Jekyll variables in CSS files!

**Comment by Scotchester** [comment]
Lookin' good! :sunglasses:

File: assets/css/main.css (hunk 1 above)
Comment (ascott1) [inline assets/css/main.css:240]: right?! I tried it on a whim and as long as you add some yaml front matter (even if it's empty) at the top of the CSS file, it works like a charm.

Comment (himedlooff) [inline assets/css/main.css:240]: Yeah you can add them to any text-based file really.

**Pull request merged**